addon/templates/components/freestyle/modal/modal-container.hbs

```diff
@@ -0,0 +1,75 @@
+{{#section.subsection name='modal Container'}}
+  {{#freestyle-collection
+    defaultKey='Basic Usage'
+    inline=false
+    as |collection|
+  }}
+    {{#collection.variant key='Basic Usage'}}
+      {{#bootstrap-button
+        classNames='btn-primary'
+        action=(action 'openModal')
+      }}
+        Open Modal
+      {{/bootstrap-button}}
+      {{#freestyle-usage 'basic-modal' name='BasicEmberBootstrapControls'}}
+        {{modal/modal-container
+          isOpen=isModalOpen
+          title="Simple Modal"
+          modalBodyContent=sampleContent
+          acceptAction=(action 'acceptModal')
+        }}
+      {{/freestyle-usage}}
+    {{/collection.variant}}
+    {{#collection.variant key='Advanced Usage'}}
+      {{#bootstrap-button
+        classNames='btn-primary'
+        action=(action 'openModal')
+      }}
+        Open Modal
+      {{/bootstrap-button}}
+      {{#freestyle-usage 'advanced-modal' name='AdvancedEmberBootstrapControls'}}
+        {{modal/modal-container
+          isOpen=isModalOpen
+          title="Simple Modal"
+          btnAcceptLabel="Ok"
+          btnAcceptStyle="success"
+          btnCloseLabel="Cancel"
+          btnCloseStyle="danger"
+          showCloseInHeader=true
+          modalBodyContent=sampleContent
+          acceptAction=(action 'acceptModal')
+          closeAction=(action 'closeModal')
+        }}
+      {{/freestyle-usage}}
+      {{freestyle-prop-types data=data propDefinitions=propDefinitions}}
+    {{/collection.variant}}
+    {{#collection.variant key='Custom Sub Components'}}
+      {{#bootstrap-button
+        classNames='btn-primary'
+        action=(action 'openModal')
+      }}
+        Open Modal
+      {{/bootstrap-button}}
+      {{#freestyle-usage 'advanced-modal' name='CustomEmberBootstrapControls'}}
+        {{#modal/modal-container isOpen=isModalOpen as |modal|}}
+          {{modal.header title="Block Modal" showCloseInHeader=true}}
+          {{#modal/-body}}
```

Comment:
Looks like this yields the body with the modal hash so it can instead be `{{#modal.body}}...{{/modal.body}}` and the documentation should be encouraging this usage.

Reply:
Agreed!